src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/aspnetcoremodulev2.wxs

```diff
@@ -27,9 +27,12 @@
 <?if $(var.Platform) = "x64" ?>
     <?define AspNetCoreV2ProgramFilesTargetPath = "$(var.AspNetCoreV2.TargetPath)" ?>
     <?define AspNetCoreV2HandlerProgramFilesTargetPath = "$(var.AspNetCoreV2Handler.TargetPath)" ?>
-<?else ?>
+<?elseif $(var.Platform) = "x86" ?>
     <?define AspNetCoreV2ProgramFilesTargetPath = "$(var.AspNetCoreV2WoW64.TargetPath)" ?>
     <?define AspNetCoreV2HandlerProgramFilesTargetPath = "$(var.AspNetCoreV2HandlerWoW64.TargetPath)" ?>
+<?elseif $(var.Platform) = "arm64" ?>
```

Comment:
Shouldn't this be❔
```suggestion
<?else ?>
```

Reply:
Shrug we only are supporting x64/x86/arm64, its sort of undefined what happens if you pass in a different platform, I almost prefer its undefined so it for sure will fail at compilation time rather than default to arm64